spec/requests/search_spec.rb

```diff
@@ -96,22 +96,45 @@
   end
 
   describe "GET /search/usernames" do
-    let(:authorized_user) { create(:user) }
-    let(:result) do
-      {
-        "username" => "molly",
-        "name" => "Molly",
-        "profile_image_90" => "something"
-      }
+    before do
+      sign_in authorized_user
     end
 
-    it "returns json" do
-      sign_in authorized_user
-      allow(Search::User).to receive(:search_usernames).and_return(
-        result,
-      )
-      get "/search/usernames"
-      expect(response.parsed_body).to eq("result" => result)
+    context "when using Elasticsearch" do
+      let(:result) do
+        {
+          "username" => "molly",
+          "name" => "Molly",
+          "profile_image_90" => "something"
+        }
+      end
+
+      it "returns json" do
+        allow(Search::User).to receive(:search_usernames).and_return(
+          result,
+        )
```

Comment:
I actually didn't style it this way so our linter/rubocop settings must've done it automatically.

When it comes to style, I typically don't bother since that's automated. For the parts that aren't automated, I tend to stick to the same as other areas of the codebase. While I do prefer your suggested styling as well since this was an automated change I'd rather leave it as is.

Reply:
Makes sense 👍🏼